securitycenter/assets/assets_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package assets
+
+import (
+	"bytes"
+	"os"
+	"testing"
+	"time"
+)
+
+func setup(t *testing.T) string {
```

Comment:
Forgot to mention, please call `testutil.SystemTest(t)` before every test starts. Can either do it here (since every test calls `setup`), or at the beginning of each test. That will skip the test if the `GOLANG_SAMPLES_PROJECT_ID` environment variable is not set.